.github/actions/setup-node/action.yml

```diff
@@ -47,11 +47,19 @@ runs:
           (steps.node-modules-restore.outputs.cache-hit == 'true') && 'true' || ''
         }}' >> "$GITHUB_ENV"
 
+    - name: Enable corepack
+      shell: bash
+      run: corepack enable
+
     - name: Setup Node
       uses: actions/setup-node@e33196f7422957bea03ed53f6fbb155025ffc7b8 # v3.7.0
       with:
         node-version: ${{ inputs.node-version }}
-        cache: ${{ env.CACHE_HIT != 'true' && 'yarn' || '' }}
+        cache: ${{ env.CACHE_HIT != 'true' && 'pnpm' || '' }}
+
+    # - name: Enable corepack (again)
+    #   shell: bash
+    #   run: corepack enable
```

Comment:
Still need this?

Reply:
I think it will otherwise use the corepack from the preinstalled nodejs on runner. 🤔

it seems to work for now, so we can remove this commented code